dotcom-rendering/src/web/components/Card/components/CardLink.tsx

```diff
@@ -13,104 +10,46 @@ const fauxLinkStyles = css`
 	right: 0;
 	bottom: 0;
 	left: 0;
-	opacity: 0;
 `;
 
-const linkStyles = (format: ArticleFormat, palette: Palette) => {
-	const baseLinkStyles = css`
-		display: flex;
-		/* a tag specific styles */
-		color: inherit;
-		text-decoration: none;
-		background-color: ${palette.background.card};
+const baseLinkStyles = css`
+	display: flex;
+	/* a tag specific styles */
+	color: inherit;
+	text-decoration: none;
+	background-color: transparent;
 
-		/* The whole card is one link so we card level styles here */
-		width: 100%;
+	/* The whole card is one link so we card level styles here */
+	width: 100%;
 
-		/* Sometimes a headline contains it's own link so we use the
+	/* Sometimes a headline contains it's own link so we use the
        approach described below to deal with nested links
        See: https://css-tricks.com/nested-links/ */
-		:before {
-			content: '';
-			position: absolute;
-			left: 0;
-			top: 0;
-			right: 0;
-			bottom: 0;
-		}
-
-		:hover .image-overlay {
```

Comment:
this rule is also present in `CardWrapper`, and I couldn't see any scenario where there would be an element with `.image-ovoerlay` inside the `CardLink`, though I may have missed something?